pipelines/deployer/deployer.yaml

```diff
@@ -202,9 +202,11 @@ apply_cluster_chart: &apply_cluster_chart
         gsp/gsp-istio-*.tgz
       function apply() {
         echo "applying ${1} from ${CHART_NAME} chart..."
-        until kubectl apply -R -f $1; do
+        kubectl apply -R -f $1 | grep -v "unchanged"
```

Comment:
We don't currently set `-o pipefail` but if we ever do this will suddenly blow up :/

Reply:
Will we still see the reason for the failure? I'm assuming it goes to stderr rather than stdout...but thought I'd check

Reply:
I think it does actually come on stdout but we'll still see that (unless it gets caught by the `grep -v` but I'm not too worried about that) - it's more that it won't do the "until" loop and will exit immediately. I think that's a bridge to cross when we come to it.